config/environments/development.rb

```diff
@@ -38,6 +42,9 @@
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
+  # Highlight code that triggered database queries in logs.
+  config.active_record.verbose_query_logs = true
```

Comment:
Was this a new default or something you added?

Reply:
Yeah so when you update rails version you always run `rails app:update` which re-generates all config files including new defaults.